src/test/kotlin/com/google/firebase/analytics/FirebaseAnalytics.kt

```diff
@@ -22,7 +25,15 @@ class FirebaseAnalytics {
     fun setCurrentScreen(currentActivity: Activity?, screenName: String?, classOverride: String?) {
         currentScreenName = screenName
     }
+    fun setConsent(var1:  MutableMap<Any, Any>) {
+        consentStateMap.putAll(var1)
+
+    }
 
+    fun getConsentState()
+    : MutableMap<Any, Any> {
+        return consentStateMap
+    }
```

Comment:
```suggestion
    }
    
```

More spacing nitpicks :P